Makefile

```diff
@@ -292,27 +311,47 @@ clean-coverage:
 
 # Building Docker Images for our executables
 ############################################
-images: user-broker-image \
-    controller-manager-image apiserver-image
+images: user-broker-image controller-manager-image apiserver-image
+
+images-all: $(addprefix sub-images-,$(ALL_ARCH))
+sub-images-%:
+	$(MAKE) ARCH=$* images
 
 define build-and-tag # (service, image, mutable_image, prefix)
 	$(eval build_path := "$(4)build/$(1)")
 	$(eval tmp_build_path := "$(build_path)/tmp")
 	mkdir -p $(tmp_build_path)
 	cp $(BINDIR)/$(1) $(tmp_build_path)
-	docker build -t $(2) $(build_path)
+	cp $(build_path)/Dockerfile $(tmp_build_path)
```

Comment:
I like that you copied the Dockerfile into "tmp" - so that we can remove the "tmp" from the paths in the Dockerfile. Much cleaner.